test/utils/mochaw.js

```diff
@@ -15,13 +15,13 @@ const asyncTestWrap = fn => done => {
   });
 }
 
-const wrapDescribeFunction = ({pageObjects},fn) => function () {
+const wrapDescribeFunction = ({screens},fn) => function () {
```

Comment:
they are still called pageobjects in the file structure, not sure if it makes clear or not :/

Reply:
Yes, good point, it was in my head but I forgot to change the folder name. Do you guys agree with this naming? If so, I will go ahead and change any reference to `pageObjects` as `screens`.

Reply:
Yeah, I agree with it. Think I ask the same question but in a different part of the code tbh.

Reply:
To me `pageObjects` seems more accurate tbh:

1. The content of the files in `pageObjects` directory is the representation of the UI **objects** on the particular **page** and behaviors/functions that use them. 

2. We follow Page Object pattern, it's good to have some indication to that, not only in the test structure but also in the naming.

Reply:
Ok, I'll put it back as `pageObjects`